registry.Dockerfile

```
@@ -5,7 +5,7 @@ WORKDIR /build
 
 COPY . .
 RUN make static
-RUN GRPC_HEALTH_PROBE_VERSION=v0.4.28 && \
+RUN GRPC_HEALTH_PROBE_VERSION=v0.4.37 && \
```

should we maybe do something like
```suggestion
RUN GRPC_HEALTH_PROBE_VERSION=$(go list -m github.com/grpc-ecosystem/grpc-health-probe | cut -d " " -f2) && \
```
?